README.md

```diff
@@ -302,7 +302,7 @@ git clone https://github.com/Microsoft/vcpkg.git
 # (consider this path as PATH_TO_VCPKG)
 cd vcpkg
 # Checkout the vcpkg commit from the vcpkg-commit.txt file (link above)
-git checkout <vcpkg commit>
+git checkout <vcpkg commit> 
```

Comment:
Revert this change :p

```suggestion
git checkout <vcpkg commit>
```

Reply:
This is like my old nemesis.  I'm going to get rid of my fork and start from scratch because this keeps sneaking back in.  :(.

In any case I'll push this and get it commited *on this PR*, even if it's a gate run for one character because I'm so sick of this and want it to stop.